### PR TITLE
manifest: Mcuboot update

### PR DESCRIPTION
**Pull request by nvlsianpu**
Mcuboot was updated to the version synchronized with
the upstream 5b7ed6a which introduces:

- Added LED support for signaling enter to the serial recovery mode
- Added boot delay/debonuce of serial detect pin.
- Fixed compilation warnings with ZEPHYR_LOG_MODE_MINIMAL
- Added optional check which prevents attempting to boot image that
has been build for different ROM address than a slot it currently
resides in. Check is enabled if image was signed with
IMAGE_F_ROM_FIXED flag.

Signed-off-by: Andrzej Puzdrowski <andrzej.puzdrowski@nordicsemi.no>

### CODE REVIEW

File: boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts [hunk 1]
```diff
@@ -120,6 +120,7 @@
 		sw1 = &button1;
 		sw2 = &button2;
 		sw3 = &button3;
+		bootloader-led0 = &led0;
```
Comment (nvlsianpu) [inline boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts:123]: @anangl Is that OK to add such aliases for all the Nordic boards like that?

File: west.yml [hunk 2]
```diff
@@ -91,7 +91,7 @@ manifest:
       revision: 24d84ecff195fb15c889d9046e44e4804d626c67
       path: modules/crypto/mbedtls
     - name: mcuboot
-      revision: 3fc59410b633a6d83bbb534e43aac43160f9bd32
+      revision: pull/46/head
```
Comment (d3zd3z) [inline west.yml:94]: zephyrproject-rtos/mcuboot#46 is ready to merge. Wouldn't we be better to just merge this, and then reference the merged result?

Comment (nvlsianpu) [reply west.yml:94]: ^^ Procedure is to check PR patch first for get CI results, then merge, change to the SHA and merge

File: boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts (hunk 1 above)
Comment (anangl) [reply boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts:123]: Does it need to be a separate alias, can't just `led0` be used?

Comment (nvlsianpu) [reply boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts:123]: MCUboot already accepted such alias for led it uses. It not necessary must be referenced to the board's led0.
Is this additional alias a problem?

Comment (anangl) [reply boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.dts:123]: I see. I should have looked at the mcuboot changes before asking...
I don't see such alias as a problem, just wondered if it should be added in Zephyr when only mcuboot uses it. But I guess it should be okay, as the alias is generic enough to be used also by other bootloaders if needed.

**Review by de-nordic — APPROVED** [review]
The "led commit" looks fine. I have already accepted the upmerge PR, awaiting the SHA update.

**Review by d3zd3z — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Review by utzig — APPROVED** [review]

**Comment by nvlsianpu** [comment]
@carlescufi Can you merge this?

**Pull request merged**